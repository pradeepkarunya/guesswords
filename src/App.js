import './App.css';
import Congrats from './congrats';
import GuessedWords from './GuessedWords';

function App() {
  return (
    <div className="container">
      <div data-test="component-app"></div>
      {/* <div>Hi</div> */}
      <Congrats success={true} />
      <GuessedWords guessedWords={[{guessedWord: 'train', letterMatchCount: 3}]} />
    </div>
  );
}

export default App;
