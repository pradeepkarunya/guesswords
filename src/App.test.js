import Enzyme, {shallow} from 'enzyme';
import EnzymeAdapter from '@wojtekmaj/enzyme-adapter-react-17';
import App from './App';

import renderer from 'react-test-renderer';


Enzyme.configure({adapter: new EnzymeAdapter()});

test('render without test',()=>{
  const wrapper = shallow(<App />);
  const el = wrapper.find('[data-test="component-app"]');
  expect(el.length).toBe(1);
});

it('renders with span message', ()=>{
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});