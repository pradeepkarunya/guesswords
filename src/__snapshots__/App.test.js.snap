// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with span message 1`] = `
<div
  className="App"
>
  <div
    data-test="component-app"
  />
  <div>
    Hi
  </div>
</div>
`;
